src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -50,7 +50,8 @@ public void periodic() {
 
     public CommandBase spinUp() {
         return new InstantCommand(() -> {
-            shooterWheelMotor.set(calculateRPM() / MAX_RPM);
+            // shooterWheelMotor.set(calculateRPM() / MAX_RPM);
```

Comment:
Please add a `TODO` comment here to note that we need to incorporate the automatic calculations again.

Reply:
Done